good-enough-practices-for-scientific-computing.tex

```diff
@@ -328,8 +329,9 @@ \subsection*{Acknowledgments}
   The best way to do this is to write scripts for \emph{every} stage of data processing.
   This might feel frustratingly slow, but you will get faster with practice. The
   immediate payoff will be the ease with which you can re-do data preparation when new
-  data arrives. You can also re-use data preparation logic in the future for related
-  projects.
+  data arrives. You can also re-use data preparation steps in the future for related
+  projects. For very large data sets, data preparation may also include scripts to obtain 
```

Comment:
"...data preparation may include writing and saving scripts to obtain the data (or subsets of the data) from remote storage."